assets/js/atomic/blocks/product-elements/summary/block.js

```diff
@@ -4,7 +4,7 @@
 import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import Summary from '@woocommerce/base-components/summary';
-import { getSetting } from '@woocommerce/settings';
+import { WORD_COUNT_TYPE } from '@woocommerce/block-settings';
```

Comment:
Let's switch back to use `getSetting` here. I'd like us to move away from using `block-settings` and use `getSetting` instead. That helps make the dependency much clearer.

This comment applies to all the other places modified to use `getSetting`.